apps/documentation/src/pages/_app.tsx

```diff
@@ -4,5 +4,9 @@ import '@voiceflow/chat/dist/style.css';
 import type { AppProps } from 'next/app';
 
 export default function App({ Component, pageProps }: AppProps) {
-  return <div>{typeof window !== 'undefined' && <Component {...pageProps} />}</div>;
```

Comment:
This wasn't needed (because we have the `clientOnly` property if we need, and it was causing me problems (i think).